less/core/notify.less

```diff
@@ -1,234 +1,115 @@
-            }
-        }
-
-        .dir-rtl & {
-            left: 0px;
-            right: inherit;
-        }
-    }
-
-    .notify-popup-icon-close {
-        color: @notify-icon-color;
-        .transition-color;
-
-        .no-touch &:hover {
-            color: @notify-icon-color-hover;
-        }
-    }
-
-    .notify-shadow {
-        background-color: @transparency;
-
-        .no-opacity & {
-            background-image: @transparency-fallback-png;
-        }
-    }
-}
-
-.notify-push {
-    position:fixed;
-    right:20px;
-    width:380px;
-    background-color: @notify-color;
-    transition:all 0.3s;
-    opacity:0;
-    margin-top:-30px;
-    color:#fff;
-    z-index:50;
-    border: 1px solid @notify-text-color;
-    border-radius:3px;
-
-    .dir-rtl & {
-        left: 20px;
-        right: inherit;
-    }
-
-    
-    @media all and (max-width:@device-width-medium) {
-        width:90%;
-        max-width:380px;
-        right:5%;
-
-	    .dir-rtl & {
-            left: 5%;
-	        right: inherit;
-	    }
-    }
-
-    &.show {
-        margin-top: 0px;
-        opacity: 1;
-    }
-}
-
-.notify-push-inner {
-    padding:30px 0px 30px 30px;
-    margin-right:(30px + @icon-size*2);
-    cursor: pointer;
+  &__popup {
+    background-color: @notify;
+    color: @notify-inverted;
+  }
+
+  &__popup-inner {
+    position: relative;
+    .l-container-padding(@notify-padding-ver, @notify-padding-hoz);
+  }
+
+  &__icon {
+    display: none;
+  }
+
+  &__title {
+    margin-bottom: @notify-title-margin;
+    .notify-title;
+
+    @media (max-width: @device-width-medium) {
+      padding-right: unit(@icon-size + @btn-margin + @btn-padding, rem);
+
+      .dir-rtl & {
+        padding-right: inherit;
+        padding-left: unit(@icon-size + @btn-margin + @btn-padding, rem);
+      }
+    }
+  }
+
+  &__body {
+    margin-bottom: @notify-body-margin;
+  }
+
+  &__instruction {
+    margin-bottom: @notify-instruction-margin;
+    .instruction-text;
+  }
+
+  &__body a,
+  &__instruction a {
+    .link-text;
+    color: @notify-link;
+
+    .no-touch &:hover {
+      color: @notify-link-hover;
+      .transition(color @duration ease-in);
+    }
+  }
+
+  &__btn-container {
+    margin-top: @btn-margin;
+  }
+
+  &__btn {
+    background-color: @notify-btn;
+    color: @notify-btn-inverted;
+
+    .no-touch &:not(.is-disabled):hover {
+      background-color: @notify-btn-hover;
+      color: @notify-btn-inverted-hover;
+      .transition(background-color @duration ease-in, color @duration ease-in;);
+    }
+  }
+
+  &__btn-icon {
+    margin: @btn-margin / 2;
+    padding: @btn-padding / 2;
+    background-color: transparent;
+    color: @notify-icon-inverted;
+    border-radius: 50%;
+
+    .no-touch &:hover {
+      background-color: @notify-icon-hover;
+      color: @notify-icon-inverted-hover;
+      .transition(background-color @duration ease-in, color @duration ease-in;);
+    }
+  }
+
+  // &__btn-icon {
```

Comment:
Should these be deleted?

Reply:
I recommend leaving comment in as code allows different style of button.